src/components/StatusUpdateForm.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 
-import Panel from './Panel';
-import PanelHeading from './PanelHeader';
-import PanelFooter from './PanelFooter';
-import PanelContent from './PanelContent';
+import {
+  Panel,
+  PanelContent,
+  PanelHeader as PanelHeading,
```

Comment:
can we skip the rename here
```suggestion
  PanelHeader,
```